WcaOnRails/app/views/competitions/_competition_info.html.erb

```diff
@@ -64,10 +64,6 @@
     <dl class="dl-horizontal">
       <dt><%= t '.information' %></dt>
       <dd><%=md competition.information %></dd>
-
-    <dl class="dl-horizontal">
-      <dt><%= t '.registration_requirements' %></dt>
-      <dd><%=md competition.registration_requirements %></dd>
     </dl>
```

Comment:
Not removing the closing tag, because I noticed there was one missing for the previous `dl`.